packages/server/lib/hooks/connection/providers/azure-devops/credentials-verification.ts

```diff
@@ -0,0 +1,14 @@
+import type { InternalNango as Nango } from '../../credentials-verification-script.js';
+
+export default async function execute(nango: Nango) {
+    const { provider_config_key } = nango.getConnection();
+
+    const response = await nango.proxy({
+        endpoint: '/_apis/projects',
```

Comment:
so it's static? can it just be an endpoint in that case? 👍🏻

Reply:
Not sure I understand what you mean

Reply:
This doesn't need to be an actual script but rather could live in the providers.yaml is a single line like this https://github.com/NangoHQ/nango/blob/master/packages/providers/providers.yaml#L11-L14

Reply:
I understand now. The reason for the script is that the success response for the permissions check is 200 while the failure response is 202. This means that both success and failure are interpreted as success, hence the need for  a script

Reply:
alright makes sense, a comment would be helpful for future reference in that case